python/lsst/source/injection/bin/make_injection_pipeline.py

```diff
@@ -120,6 +120,32 @@ def build_argparser():
         help="Config override for a task, in the format 'label:key=value'.",
         action="append",
     )
+    parser.add_argument(
+        "-ip",
+        "--input-pipelines",
+        type=str,
+        help="""Location(s) of additional input pipeline definition YAML
+        file(s). Tasks from input_pipelines will be added to the injection
+        pipeline.""",
+        required=True,
+        metavar="FILE",
+        nargs="+",
+    )
+    parser.add_argument(
+        "-sn",
+        "--subset_name",
+        type=str,
+        help="""All tasks from input_pipelines will be added to this subset.
```

Comment:
"All tasks from input_pipelines will be..." -> "All tasks from any additional pipelines will be..."